R/resultAnnotations.R

```diff
@@ -0,0 +1,925 @@
+#' 
+#' @title Additional result annotations
+#' 
+#' @description These functions work on the result table and add additional 
+#'     annotations to the reported introns: the type of splice event (e.g. 
+#'     exon skipping, exon truncation, ...), expected occurence of frameshift, 
+#'     overlap with UTR regions as well as a flag for introns that are 
+#'     located in blacklist regions of the genome.
+#'     
+#' \code{\link{annotateIntronReferenceOverlap}} adds basic annotations to the 
+#'     fds for each intron based on the overlap of the intron's location with 
+#'     the reference annotation. Has to be run before the result table is 
+#'     created so that the new column can be included in it (see examples).
+#' 
+#' \code{\link{annotateSpliceEventType}} annotates each intron in the results 
+#'     table with the type of splice event and expected occurence of frameshift 
+#'     (likely, unlikely, inconclusive). Can also calculate overlap with 
+#'     annotated UTR regions. Splice types can be one of: 
+#'     annotatedIntron_increasedUsage, annotatedIntron_reducedUsage, 
+#'     exonTruncation, exonElongation, exonTruncation&Elongation, 
+#'     singleExonSkipping, exonSkipping, splicingBeyondGene, 
+#'     multigenicSplicing, downstreamOfNearestGene, upstreamOfNearestGene, 
+#'     complex (everything else).     
+#'     Splice sites (theta metric) annotations indicate how the splice site is 
+#'     located with respect to the reference annotation. The annotated types 
+#'     are: annotatedSpliceSite, exonicRegion, intronicRegion.
+#' 
+#' \code{\link{flagBlacklistRegions}} flags introns in the results table on 
+#'     whether or not they are located in a blacklist region of the genome. By
+#'     default, the blacklist regions as reported in \cite{...} are used.
```

Comment:
Please fill in the cite part so we know where it comes from. If we use a file we downloaded maybe rather use a link to the download page.